### DISCUSSION TITLE
Privacy Policy in GooglePlay

### DISCUSSION THREAD

**Discussion by Tywors** [journeyapps/zxing-android-embedded#604] [Q&A]
I add zxing in my android app and now googlePlay say that is mandatory add a privacy policy document because i add a Camera permission.

I don't have a camera permission, is zxing.

What document can i add to googlePlay?
Thanks

**Comment by rkistner** [comment]
The camera is used to scan barcodes, nothing more.

The privacy policy is completely up to you. You may have to get legal advice on that, and it's completely outside the scope of this project.

**Comment by klemensz** [comment]
The `android.permission.CAMERA` will be added by this library, even if the app doesn't use it to scan barcodes. In our case for example we use it just do display barcodes/QR codes. When uploading the app to Google Play Store it will show a warning "... requesting unnecessary permissions can affect your app's visibility on the Play Store".

**Comment by rkistner** [comment]
You can remove the permission by adding this to your manifest:

```xml
<uses-permission android:name="android.permission.CAMERA" tools:node="remove"/>
```

That said, the main purpose of this library is the camera control. If you're only using this library for encoding barcodes, you could also just copy the source of single class for that, and depend on `com.google.zxing:core` directly: https://github.com/journeyapps/zxing-android-embedded/blob/master/zxing-android-embedded/src/com/journeyapps/barcodescanner/BarcodeEncoder.java
